src/command/global/start.ts

```diff
@@ -20,7 +20,7 @@ export const handler = async (interaction: CommandInteraction) => {
     const settingsRepository = getCustomRepository(
         NotificationSettingsRepository
     );
-    const apiKey = interaction.options.getString("apikey");
+    const apiKey = interaction.options.getString("apikey", true);
     if (apiKey) {
```

Comment:
this null check is no longer needed. The function above throws if it's null (which it can't be by requirement)